accord-core/src/main/java/accord/local/Node.java

```diff
@@ -70,10 +73,16 @@ public String toString()
         }
     }
 
+    public static int numCommandShards()
+    {
+        // TODO: make configurable
+        logger.warn("TODO: using hardcoded value for number of command shards, make configurable");
```

Comment:
I think it's fine to leave the simple code TODO, we're still very much in the prototyping and design phase, and won't forget this